libwarpaffine/sliceswriter/ISlicesWriter.h

```diff
@@ -63,6 +63,10 @@ class ICziSlicesWriter
     /// Adds a slice (or subblock).
     /// \param  add_slice_info Information describing the add slice.
     virtual void AddSlice(const AddSliceInfo& add_slice_info) = 0;
+    
+    /// @brief Adds a attachment, i.e. a copy from a source document.
+    /// @param attachment The attachment to be added.
```

Comment:
Suggest using the documentation style used by the existing code - so this would be
```suggestion
    /// Adds an attachment, i.e. a copy from a source document.
    /// \param attachment The attachment to be added.
```